extension/persistence/relational-jdbc/src/main/java/org/apache/polaris/extension/persistence/relational/jdbc/JdbcBasePersistenceImpl.java

```diff
@@ -86,16 +87,64 @@ public void writeEntity(
       @Nonnull PolarisBaseEntity entity,
       boolean nameOrParentChanged,
       PolarisBaseEntity originalEntity) {
+    persistEntity(callCtx, entity, originalEntity, datasourceOperations);
+  }
+
+  @Override
+  public void writeEntities(
```

Comment:
nit: this is a major change to `JdbcBasePersistenceImpl`, but the PR says it's only about "plumbing" :shrug:

Reply:
I agree, one way to look at this this plumbing exposed implemention bug which could only be caught by service test hence this is a fix

Reply:
It's fine to fix, but please mention it in the PR summary :)